WcaOnRails/app/assets/javascripts/persons.js

```diff
@@ -15,11 +15,6 @@ onPage('persons#index', function() {
   var $table = $('.persons-table');
   var options = $table.bootstrapTable('getOptions');
 
-  // Set the table options from the url params.
-  options.pageNumber = parseInt($.getUrlParam('page')) || options.pageNumber;
```

Comment:
That was very helpful. I learned a lot! I thought I had tested exactly that `page=2` scenario, but reality strongly disagrees.

I think this new code gets the job done.